src/Components/FieldDisplay.tsx

```diff
@@ -61,7 +61,10 @@ export function NodeFieldDisplay({ node, oldValue, suffix, component }: { node:
   const formulaTextOverlay = !!node.formula && <QuestionTooltip title={<Typography><Suspense fallback={<Skeleton variant="rectangular" width={300} height={30} />}>{fieldFormulaText}</Suspense></Typography>} />
   return <Box width="100%" sx={{ display: "flex", justifyContent: "space-between", gap: 1 }} component={component} >
     <Typography color={`${node.info.variant}.main`} sx={{ display: "flex", gap: 1, alignItems: "center" }}>{!!isTeamBuff && <Groups />}{icon}{fieldText}{suffix}</Typography>
-    <Typography sx={{ display: "flex", gap: 1, alignItems: "center" }} >{fieldVal}{formulaTextOverlay}</Typography>
+    <Box sx={{ display: "flex", gap: 1, alignItems: "center" }}>
+      <Typography noWrap>{fieldVal}</Typography>
```

Comment:
noWrap will make it so that extra text gets `...`'d

Reply:
I know the API says that, but I wasn't able to get that to happen even in the most extreme cases (smallest column, with an impossibly large number on Yae's skill)
![image](https://user-images.githubusercontent.com/36019388/172075429-cc5acc69-2473-4d4d-82a1-524d05b5a76a.png)

Reply:
Speaking about this particular change, do we need to wrap this in FieldDisplayList or similar, so we can get alternating row colors for clarity?
![image](https://user-images.githubusercontent.com/36019388/172075616-ac259d05-f341-4a62-911a-6c795d9f1127.png)

Reply:
i guess we can, at one point that alternating color was conflicting with the background, but now i dont think there is an instance

Reply:
I'll add it in tonight. After that, should be good to go